governance/roles.md

```diff
@@ -0,0 +1,96 @@
+# Roles within the TAG App Delivery
+
+The TAG includes several key roles that are critical to the group's success.
+Beyond the roles described in this document the TAG has many contributors, all serving in varying capacities.
+
+- [Roles within the TAG App Delivery](#roles-within-the-tag-app-delivery)
+    - [Chair](#chair)
+    - [Technical lead](#technical-lead)
+    - [Working Group Lead](#working-group-lead)
+    - [TOC Liaison](#toc-liaison)
+    - [Emeritus Chair](#emeritus-chair)
+
+## Chair
+
+The (normally three) Co-Chairs of the TAG App Delivery drive the CNCF community efforts of the TAGs domain.
```

Comment:
How about expressing this as "the TAG will have up to 3 chairs"? Best to be specific since folks will look to this as canonical in the future.